yarn-project/ethereum.js/src/contract/bytes.ts

```diff
@@ -1,33 +1,236 @@
+/**
+ * The Bytes class extends the built-in Buffer class in Node.js, providing a convenient and efficient way
+ * to work with binary data. It is designed for use in various scenarios such as reading/writing files,
+ * network communication, and cryptographic operations. Each Bytes subclass (Bytes1 to Bytes32) represents
+ * a specific fixed-length byte array, allowing for optimized memory management and type safety.
+ */
 export class Bytes extends Buffer {}
+/**
```

Comment:
There's many ByteN classes, can you replace these all with the line
`// eslint-disable-next-line jsdoc/require-jsdoc`?